modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -300,22 +350,26 @@ Diagram.defaultProps = {
 			selectedColor: '0B5FFF',
 			useNative: true,
 		},
-		defaultRadius: 15,
+		defaultRadius: 2,
 	},
 	pins: [],
+	pinsEndpoint:
+		'http://localhost:8080/o/headless-commerce-admin-catalog/v1.0/',
```

Comment:
🤔  is this supposed to point to this url or something not local? Is it like a Docker image or something that we can rely on existing?

Reply:
This is a defaultProps only to be sure it will not be empty.I guess this endpoint links always to the local instance where catalog's api are working. Can be possible to add a different endpoint just passing in a different jsp-template a different string like that we have in our diagram.jspf `		).put(
			"pinsEndpoint", "/o/headless-commerce-admin-catalog/v1.0/"
		).put(`

Reply:
Yeah we shouldn't be committing code that is referencing `localhost`. If we need a default prop for this, either use the relative path or combine with url root from `window.location` object